src/traces/bar/plot.js

```diff
@@ -67,9 +67,12 @@ module.exports = function plot(gd, plotinfo, cdModule, traceLayer, opts) {
     var bartraces = Lib.makeTraceGroups(traceLayer, cdModule, 'trace bars').each(function(cd) {
         var plotGroup = d3.select(this);
         var trace = cd[0].trace;
+        var isWaterfall = (trace.type === 'waterfall');
+        var isFunnel = (trace.type === 'funnel');
+        var isBar = !(isWaterfall || isFunnel);
```

Comment:
Ok, so `isBar` includes histogram traces. This looks ok, as I don't think you can get a zero-height bar in histogram to go through here. But I think it would be safer to that have this as `trace.type === 'bar'`.

Reply:
Good call. 
Done in 97190c3.